src/libraries/Common/tests/System/Drawing/Helpers.cs

```diff
@@ -50,7 +54,7 @@ public static bool GetIsWindowsOrAtLeastLibgdiplus6()
                 return false;
             }
 
-            return installedVersion.Major >= 6;
+            return installedVersion >= version;
```

Comment:
Versions have interesting comparisons when parts of the version aren't specified. It would probably be better to just take major/minor and explicitly check.